src/app/notes/notes.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="flex">
+  <app-note-list
+    class="list-component"
+    [noteList]="noteList"
+    [displayedColumns]="displayedColumns"
+    (addNote)="noteService.addNote()"
```

Comment:
Personally, I'd pull out these service calls into functions in the component. So like (addNote)="noteAdded()"

And then noteAdded() would call this.noteService.addNote()

Reply:
Same with editNote